instagram/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import './App.css';
+import './DummyData/dummy-data';
+import dummyData from './DummyData/dummy-data';
+import PostContainer from './components/PostContainer/PostContatiner';
+import SearchBar from './components/SearchBar/SearchBar';
+
+
+class App extends Component {
+  constructor(){
+    super();
+    this.state={
+      data : dummyData,
+    }
+  }
+  render() {
+    return (
+      <div className="App">
+        <SearchBar />
+        <PostContainer dataProps = {this.state.data} />
```

Comment:
so long as you reference it in PostContainer this is fine - -but for clarity's sake and to avoid potential confusion, I recommend having the same names for the piece of data being passed in both parent and child component, so `data={this.state.data}`. that way you're always referencing it with the same name in all locations within your app